### ISSUE TITLE
I can't launch Nintendo Switch Sports on Yuzu

### ISSUE THREAD

**Issue by PaolaGith** [yuzu-emu/yuzu#8704]
Hi,
I found that other users had this problem, but the issue was closed. Was the problem actually solved?
Because, I am still not able to play it. How did you manage to solve? What should I do?

Thanks a lot!

**Comment by german77** [comment]
This game is not playable yet. Only boots on nvidia with latest firmware. Even then you have severe graphical issues

**Comment by XingKongSync** [comment]
Same problem here, I can only see a intro, when I press 'A', the emulator will crash after a few seconds

**Comment by smoneta** [comment]
I tested yesterday, the latest public version of yuzu and game + update... it launches and it works but the graphics still are glitchy .. like in tennis everything is black and you can only see the characters or in bowling, the images drag like when the images freeze on cable tv and the next images drag... or when you win windows solitary.

**Comment by german77** [comment]
All mini games work. But when they look like this it's not worth calling it playable. For everyone who can't boot the game is due lack of vram. This game only work if you have more than 2GB probably due the same gpu errors.
https://user-images.githubusercontent.com/5944268/186206140-2fd5fd0a-413d-43e8-ab11-00845fcc52c6.mp4

**Comment by MopheusDG** [comment]
I agree with @german77 game boots ok for me, just has too many graphical issues (lost textures I think, etc). Would love to see this one running.

Thanks Yuzu team.

**Comment by TomMettam** [comment]
Would love to see this working. It's so close! fully functional just with corrupt sprites.

**Comment by MopheusDG** [comment]
Hi @TomMettam , but is NV_shader_buffer_store already implemented in OpenGL ? I mean, should I try with my GTX1070 the game on OpenGL to see what happens ?

Thanks.

**Comment by TomMettam** [comment]
I'm afraid I'm just an interested party and not a Yuzu dev, so I don't know. My interest is yuzu on steamdeck (AMD)

**Comment by Curiosity007** [comment]
Earlier build was able to run the game with glitched graphics. With build 1303, this game exits completely from the main screen, and is not starting.

**Comment by StevphenNew** [comment]
It's 2023 Aug 26 but I can still not open it in Vulkan(crash after A button).  Opengl is able to play,  but the character and main scene in every sport are all black

**Comment by salomaoparkour** [comment]
I too have the same issue as @StevphenNew.
Dolphin had the same flaw when it came to Wii Sports Resort. They were never able to make it fully work.

I wonder if the Nintendo Sports challenge is so hard that the developers just ignore it.
